sonar-slang-plugin/src/main/java/com/sonarsource/slang/kotlin/CommentAndTokenVisitor.java

```diff
@@ -51,6 +56,13 @@
   private static final int DOC_COMMENT_SUFFIX_LENGTH = 2;
   private static final int LINE_COMMENT_PREFIX_LENGTH = 2;
 
+  private static final Set<KtToken> LITERAL_TOKEN_TYPES = new HashSet<>(Arrays.asList(
+    KtTokens.INTEGER_LITERAL,
+    KtTokens.FLOAT_LITERAL,
```

Comment:
Are you sure that `INTEGER_LITERAL` and `FLOAT_LITERAL` should be mapped to `"LITERAL"` too? In other analyzers (Swift / TSQL) I have only seen the strings mapped to `"LITERAL"`

Reply:
Fixed in 097155a